sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaigns/CampaignGrid.java

```diff
@@ -85,7 +86,10 @@ public CampaignGrid(CampaignCriteria criteria) {
 
 		for (Column<?, ?> column : getColumns()) {
 			column.setCaption(I18nProperties.getPrefixCaption(CampaignIndexDto.I18N_PREFIX, column.getId(), column.getCaption()));
+			column.setMinimumWidth(500);
```

Comment:
This now makes almost all columns way too wide. Only the two icon columns need a minimum width to make sure that the heading is still shown because the icon is too small to ensure that.

Also, please rename the "Open" column to "View" because it's not technically used to open the campaign, but to view its data.

Reply:
@MartinWahnschaffeSymeda told me to name the column "Open"

Reply:
Yes, I'd prefer open, because we don't want to view the campaign itself, but to open it's campaign data :-D

Reply:
Okay, fine by me then!